tests/ui/test_ui_tools.py

```diff
@@ -1397,7 +1397,15 @@ def test_private_message_to_self(self, mocker):
         ('<hr/>', ['[RULER NOT RENDERED]']),
         ('<img>', ['[IMAGE NOT RENDERED]']),
         ('<img/>', ['[IMAGE NOT RENDERED]']),
-        ('<table>stuff</table>', ['[TABLE NOT RENDERED]']),
+        ('<table><thead><tr><th>Firstname</th><th>Lastname</th></tr></thead>'
+         '<tbody><tr><td>John</td><td>Doe</td></tr><tr><td>Mary</td><td>Moe'
+         '</td></tr></tbody></table>', [
+            '┌───────────┬──────────┐\n',
+            ('bold', '│ Firstname │ Lastname │\n'),
+            '├───────────┼──────────┤\n',
+            '│ John      │ Doe      │\n',
+            '│ Mary      │ Moe      │\n',
+            '└───────────┴──────────┘']),
```

Comment:
You only have one test here, but support at least different alignments, of which there is no indication here?